src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs

```diff
@@ -191,6 +192,48 @@ public static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command)
             return scope;
         }
 
+        public static Scope CreateRabbitMQScope(Tracer tracer, out RabbitMQTags tags, string command, ISpanContext parentContext = null, string spanKind = SpanKinds.Client, DateTimeOffset? startTime = null, string queue = null, string exchange = null, string routingKey = null)
```

Comment:
Shouldn't we use `SpanKinds.Producer` and `SpanKinds.Consumer` for RabbitMQ? Too late to change now until Tracer 2.0?

Reply:
Yeah we already use `SpanKinds.Producer` and `SpanKinds.Consumer`. Here's the spanKind for each operation:
- Consumers use `SpanKinds.Consumer`: BasicDeliver, BasicGet
- Producers use `SpanKinds.Producer`: BasicPublish
- Queue/Exchange operations default to `SpanKinds.Client`: ExchangeDeclare, QueueBind, QueueDeclare

So, nothing needs to change here, though I like your suggestion to make spanKind a required parameter.

Reply:
Thanks. I should have looked at the code that calls into this. I just saw the default `SpanKinds.Client` and assumed we were only using client & server.